### PR TITLE
fix: assets depends on config settings

### PR DESCRIPTION
**Pull request by dni**
now asset select will actually use assets configured on config.js

also closes https://github.com/BoltzExchange/boltz-web-app/issues/344

### CODE REVIEW

**Review by kilrau — CHANGES_REQUESTED** [review]
Works on mainnet, on testnet it still happens (see url, it's the testnet deployment from this branch)

image

File: src/components/AssetSelect.jsx [hunk 1]
```diff
@@ -49,6 +50,14 @@ const SelectAsset = () => {
         );
     };
 
+    const assets = () => {
+        const names = Object.keys(pairs).map((pair) => {
+            return pair.split("/")[0];
+        });
+        names.push(LN);
+        return names;
+    };
```
Comment (michael1011) [inline src/components/AssetSelect.jsx:59]: Why is this a function? It does only ever need to be evaluated once. Configs don't do reactive changes.

Comment (michael1011) [inline src/components/AssetSelect.jsx:59]: Why is this a function? It does only ever need to be evaluated once. Configs don't do reactive changes.

**Review by michael1011 — CHANGES_REQUESTED** [review]

**Review by michael1011 — APPROVED** [review]

**Review by kilrau — CHANGES_REQUESTED** [review]
Can you rebase this on master so the `C.json` fix is in and I can give this another proper test? @dni

**Comment by dni** [comment]
rebased

**Review by michael1011 — APPROVED** [review]

**Pull request merged**